lagom13-scala-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/lagom/scaladsl/ConductRApplicationComponents.scala

```diff
@@ -40,7 +40,7 @@ trait ConductRServiceLocatorComponents extends BundlelibComponents with LagomSer
   def actorSystem: ActorSystem
   def configuration: Configuration
 
-  lazy val circuitBreakerMetricsProvider: CircuitBreakerMetricsProvider = new CircuitBreakerMetricsProviderImpl(actorSystem)
+  override lazy val circuitBreakerMetricsProvider: CircuitBreakerMetricsProvider = new CircuitBreakerMetricsProviderImpl(actorSystem)
```

Comment:
Hmmm. Perhaps 1.3.5 should have been a 2.0.0 as it breaks compatibility, no? I'd prefer to stick with 1.3.5 and force people to upgrade to that.